### ISSUE TITLE
Stop writing Modem->Dev in Pair

### ISSUE THREAD

**Issue by krkeegan** [TD22057/insteon-mqtt#249]
This entry is required for i2cs devices and it is what the Join command creates.  If we are already able to communicate with the device, then this link is already there, yet the Pair command always tries to add it.  I think we should stop doing that, it just adds more text for the user, it risks that we somehow change the Join link in a way that doesn't make things happy, and if the device isn't i2cs this is just more work that doesn't need to be done.

This isn't really a bug, just a way to improve/streamline things.

**Comment by krkeegan** [comment]
Here is an example of why this is silly

This is a mini remote that I factory refreshed.  Then ran Join then Pair and the resulting relevant part of the DB
```
  0ff7: 41.ee.e6 (modem)          grp:   1 type: RESP data: 0xff 0x00 0x01
  0fff: 41.ee.e6 (modem)          grp:   1 type: RESP data: 0x03 0x00 0x00
```

The first Resp record was created by the Join command, and the second by our pair command.  There is no reason for there to be two entries.

This could have been caught if our link_data for responder happened to match for the remote, but the remote doesn't have a need to be accurate for responders since it doesn't respond.  Plus, it isn't clear that we would get the data_3 value right.

**Comment by krkeegan** [comment]
This change was implemented in #277
